apps/deploy-web/src/components/sdl/EnvVarList.tsx

```diff
@@ -51,8 +55,8 @@ export const EnvVarList: React.FunctionComponent<Props> = ({ currentService, set
         </span>
       </div>
 
-      {(currentService.env?.length || 0) > 0 ? (
-        currentService.env?.map((e, i) => (
+      {(currentEnvs?.length || 0) > 0 ? (
```

Comment:
this should be sufficient. unless I'm missing smth
```suggestion
      {currentEnvs?.length ? (
```